apps/src/aichat/views/modelCustomization/SetupCustomization.tsx

```diff
@@ -78,25 +79,38 @@ const SetupCustomization: React.FunctionComponent = () => {
   const renderChooseAndCompareModels = () => {
     return (
       <div className={styles.inputContainer}>
-        <SimpleDropdown
-          labelText="Selected model:"
-          onChange={event =>
-            dispatch(
-              setAiCustomizationProperty({
-                property: 'selectedModelId',
-                value: event.target.value,
-              })
-            )
-          }
-          items={availableModels.map(model => {
-            return {value: model.id, text: model.name};
-          })}
-          selectedValue={chosenModelId}
-          name="model"
-          size="s"
-          className={styles.selectedModelDropdown}
-          disabled={isDisabled(selectedModelId) || readOnlyWorkspace}
-        />
+        <WithTooltip
+          tooltipProps={{
+            text: 'This is the underlying language model being used by the chatbot. Use the dropdown to select from additional fine-tuned models.',
+            size: 's',
+            tooltipId: 'selected-model-tooltip',
+            direction: 'onLeft',
+            className: styles.tooltip,
+          }}
+          tooltipOverlayClassName={styles['full-width']}
+        >
+          <div className={styles['full-width']}>
+            <SimpleDropdown
+              labelText="Selected model:"
```

Comment:
Just curious if there is a way we can add a tooltip to the label text and not both the label text and ropdown. Thanks!
cc: @levadadenys

Reply:
Updated this to hide the default label, so we can display a separate label element with the icon. That way the tooltip doesn't appear when hovering the actual dropdown.